### PR TITLE
#287 Update color and gradient in Mission/Vision

### PR DESCRIPTION
**Pull request by aleksandarmicev**
Changed color for: 1.Gradient, 2. Color of Mission/Vision text card

### CODE REVIEW

File: components/module-components/Mission&Vision/missionVision.module.scss [hunk 1]
```diff
@@ -139,7 +135,7 @@
 }
 
 .missionTitleDark {
-  background: rgba(54, 67, 98, 1);
+  background: var(--Primary, hsla(222, 50%, 14%, 1));
```
Comment (edichoska) [inline components/module-components/Mission&Vision/missionVision.module.scss:138]: could you update the colors file as well? (colors.scss) and add this values as well so we can reuse them through out the project

File: app/styles/globals/_colors.scss [hunk 2]
```diff
@@ -19,61 +19,4 @@
...
-  $clr-secondary-light-mode: hsl(31, 94%, 86%);
-  $clr-tertiary-light-mode: hsl(222, 50%, 14%);
-  $clr-text-light-mode: hsl(215, 19%, 35%);
-  $clr-elevated-elements-light-mode: hsl(210, 40%, 98%);
-  $clr-titles-light-mode: hsl(222, 50%, 14%);
-  $clr-tertiary-button-light-mode: hsl(0, 0%, 0%);
-
-  $clr-primary-dark-mode: #121d36;
-  $clr-secondary-dark-mode: hsl(222, 19%, 44%);
-  $clr-tertiary-dark-mode: hsl(222, 52%, 5%);
```
Comment (edichoska) [inline app/styles/globals/_colors.scss:33]: @aleksandarmicev  Ace is this a mistake or on purpose? I see you have deleted all the colors from colors.scss

Comment (aleksandarmicev) [reply app/styles/globals/_colors.scss:33]: It's not a mistake, I purpose in the code to exist separate file for color variables, and in color to be only primary colors

Comment (edichoska) [reply app/styles/globals/_colors.scss:33]: why would we do that?

Comment (edichoska) [reply app/styles/globals/_colors.scss:33]: Please undo that.
This ticket is only for updating the colors used in the Mission/Vision section.
We can discuss updating the Sass file structure. Maybe we can use the 7:1 method. But we should create a separate issue for that. 🙏 
Try to keep the PRs only about what was mentioned in the ticket.

Comment (aleksandarmicev) [reply app/styles/globals/_colors.scss:33]: Ok, I'll revert these changes.

Comment (edichoska) [reply app/styles/globals/_colors.scss:33]: Thank you

**Review by edichoska — APPROVED** [review]
🚀

**Pull request merged**